src/_variables.scss

```diff
@@ -702,8 +711,8 @@ $navbar-light-toggler-border-color: rgba($black, .1) !default;
 
 // Fluid Navbar
 
-$navbar-width:                      $font-size-base * 20.0;
-$navbar-width-sm:                   $font-size-base * 5.0;
+$navbar-width:                      px(200);
```

Comment:
What do you think about prefixing our own variables? Eg.: `$fds-navbar-width`

Reply:
The fluid-navbar stuff above is actually only colors and it's in the Colors section. I think the only variables with a "fluid" prefix are some of the colors, and I only did that because those colors needed names. Like we need some way to indicate what "fluid-green" is and how it is different from "green", so I named it "fluid-green". It's not really a prefix, it's just the name of the color. For variables where we're not introducing an alternative to an existing Bootstrap variable, I didn't feel the need to prefix them. I don't know, I'm on the fence about it.